tests/src/test/scala/com/nvidia/spark/rapids/udf/URLDecode.scala

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package com.nvidia.spark.rapids.udf.scala
+package com.nvidia.spark.rapids.udf
```

Comment:
I assume this is getting moved to avoid colliding with the "real" UDFs in the udf-examples module, so since we're moving, let's use a package name to make it clear these are just test UDFs and not intended for real use, e.g.: `com.nvidia.spark.rapids.test.udf`, `com.nvidia.spark.rapids.udf.test`, or something similar.